packages/suite/src/views/wallet/anonymize/index.tsx

```diff
@@ -17,7 +17,9 @@ const Anonymize = () => {
     return (
         <WalletLayout title="TR_NAV_ANONYMIZE" account={selectedAccount}>
             <WalletLayoutHeader title="TR_NAV_ANONYMIZE">
-                <StyledAnonymityIndicator />
+                <AnynomityLevelSetupWrapper>
```

Comment:
ok, but lets discuss on next Code Improvements :-)

Reply:
for clarity: I meant, style the child – ` <AnonymityLevelSetup />` instead of wrapping it with another component just for a margin. And what would you like to discuss?

Reply:
I get it, and I'm not sure what is better in general. For one thing, I don't like all the names start with `Styles...`, other thing is whether we should prefer extending styles of existing component or creating a wrapper for layout styles
https://github.com/trezor/trezor-suite/pull/6569/commits/ad015a81bb1912f94943f99f436d394fb3cb2e85

Reply:
I see!